.travis.yml

```diff
@@ -176,6 +176,8 @@ matrix:
       if: branch = auto
     - env: IMAGE=x86_64-gnu-distcheck
       if: branch = auto
+    - env: IMAGE=mingw-check
+      if: type = pull_request
```

Comment:
I'd actually run this on the `auto` branch as well just to be extra sure it's always passing